src/07-projects/project-items.apib

```diff
@@ -0,0 +1,63 @@
+## Project items [/projectItems]
+
+Project items are added to a project and they affect the profitability. This information is only available for account administrators and project decision makers.
+
+*Required scopes: `projects`*
+
+### projectItems.list [GET /projectItems.list]
+
+Get a list of project items.
+
++ Request (application/json;charset=utf-8)
+
+    + Attributes (object)
+        + filter (object, optional)
+            + project_id: `082e6289-30c5-45ad-bcd0-190b02d21e81` (string, optional)
+        + page (Page, optional)
+        + sort (array, optional)
+            + (object)
+                + field (enum[string], required)
+                    + Members
+                        + date
+                + order (enum[string], optional)
+                    + Members
+                        + asc
+                        + desc
+            + Default
+                + (object)
+                    + field: `date`
+                    + order: `desc`
+
+
++ Response 200 (application/json;charset=utf-8)
+
+    + Attributes (object)
+        + data (array)
+            + (object)
+                + id: `cfb4146d-06be-41f1-bb39-aa3c929c71dc` (string)
+                + project (object)
+                    + type: `project` (string)
+                    + id: `944534fb-15f1-4eea-aab1-82a427aa2d0d` (string)
+                + milestone (object, nullable)
+                    + type: `milestone` (string)
+                    + id: `944534fb-15f1-4eea-aab1-82a427aa2d0d` (string)
+                + type: `time_tracked` (enum[string])
+                    + Members
+                        + time_tracked
+                        + external_cost_item
+                + description: `Drafted the document` (string, nullable)
+                + date: `2018-11-26T16:44:33+00:00` (string)
+                + employee (object, nullable)
```

Comment:
As we try to avoid the term `employee` I think we should try to find another name. /cc @stivni @jenssegers

Reply:
I think employee is the correct term here. It references the employee who did the time that was tracked, or the employee that added the order item.

Reply:
But we only want to use the term `employee` in HR context. That is why I reckon we should try to find another name.

Reply:
we didn't get this straight enough yet, indeed...
can I propose to wait until the domain discovery workshop before we coin the terms?

Reply:
Ok, updated to participant.